packages/components/constraints/src/helpers.js

```diff
@@ -0,0 +1,35 @@
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const getMaxPropTokenValue = (max) => {
+  if (max === 1) return vars.constraint1;
```

Comment:
I think this will be tedious to update later.
can this be 

```js
// ensure null, not undefined
return vars[`constraint${max}`] ?? null
```

it's just an idea. I don't mind that we keep this as is.

Reply:
Yeah it works nicely for the current scale system 👍